### PR TITLE
feat: support for Ed25519 server keys for signing the JWT tokens

### PR DESCRIPTION
**Pull request by tranchitella**
Ticket: MEN-5676
Changelog: Title

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline --pr deviceauth/674 --fast

File: config.yaml [hunk 1]
```diff
@@ -12,11 +12,23 @@ listen: :8080
 # Defaults to: prod
 # middleware: dev
 
+# Private key type - used for JWT signing
+# Defaults to: rsa
+# Supported values: rsa, ed25519
+# Overwrite with environment variable: USERADM_SERVER_PRIV_KEY_TYPE
+# server_priv_key_type: rsa
```
Comment (alfrunes) [inline config.yaml:19]: Same as for the deviceauth PR, this option is redundant and could be resolved by proper parsing of the PKCS#8 structure.

Comment (tranchitella) [reply config.yaml]: done

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline --pr deviceauth/674

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline --pr deviceauth/674

**Pull request merged**